scripts/index.js

```diff
@@ -452,7 +526,11 @@ program
     .option('-d, --debug', 'debug visual tests by overriding refs (golden images) in case of visual changes', false)
     .option('-r, --recreate', 'recreate visual refs (golden images)', false)
     .option('-v, --verbose', 'log passing tests', false)
-    .option('-cc, --clear-cache', 'clear CLI test cache', false)
+    .option('-l, --launch', 'launch tests in the browser', false)
+    .option('--ci, --no-ci', 'runs testem with a `ci` flag', true)
```

Comment:
perhaps a better options would be `--dev` that is the opposite of `--ci`

Reply:
[f54ee83](https://github.com/fabricjs/fabric.js/pull/8138/commits/f54ee831cdbcbf681d66ff9a4abb73e8c749fe8a)